packages/components/src/components/select/select/select.tsx

```diff
@@ -151,6 +151,7 @@ export class Select {
   private isNativePopoverCase: boolean = false;
   private parentTableElement: HTMLElement;
   private popoverElement: HTMLElement;
+  private hasSlottedImage: boolean = false;
```

Comment:
Why not directly save the current img src here and reuse in the render instead of calling `this.getSelectedOptionImagePath` again?